README.rst

```diff
@@ -208,6 +212,21 @@ where:
 
 .. _`transmart v2 api constraint`: https://github.com/thehyve/transmart-core/blob/dev/open-api/swagger.yaml
 
+
+Adding new entity to CSR data model:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When the CSR data model is extended with new entities, the export transformation code
+has to be changed as well in order to include a column with the ID of the new entity as one of the identifying columns.
+
+In order to do this, `<table_transformations/csr_transformations.py>`_ file has to be modified.
+Required changes:
+
+1) extend the `ID_COLUMNS` list,
+
+2) modify `from_obs_df_to_csr_df(obs: DataFrame)` function in order to properly rename columns.
```

Comment:
Inline code in rst is done with double backticks: ` ``from_obs_df_to_csr_df(obs: DataFrame)`` `